### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by donizer**
[DEMO LINK](https://donizer.github.io/react_dynamic-list-of-goods/)

### CODE REVIEW

**Comment by donizer** [comment]
There's error on tests. I fixed it locally executing `npm i` two times. Don't know what to do here.

File: src/index.tsx [hunk 1]
```diff
@@ -1,6 +1,9 @@
+import React from 'react';
 import ReactDOM from 'react-dom';
 import { App } from './App';
 
+React.createElement('div');
```
Comment (VitaliyBodnar) [inline src/index.tsx:5]: why do you need that ?
In my opinion it's redundant

File: src/GoodsList.tsx [hunk 2]
```diff
@@ -7,10 +7,16 @@ type Props = {
...
-    ))}
+    {goods.map(good => {
+      const style: React.CSSProperties = {
+        color: good.color,
+      };
+
+      return (
+        <li key={good.id} data-cy="good" style={style}>
+          {good.name}
+        </li>
```
Comment (VitaliyBodnar) [inline src/GoodsList.tsx:18]: don't see any sense to create a separate object for only one style 
```suggestion
      return (
        <li 
           key={good.id} 
           data-cy="good" 
           style={{ color: good.color }}
          >
           {good.name}
        </li>
```

**Review by VitaliyBodnar — CHANGES_REQUESTED** [review]

File: src/index.tsx (hunk 1 above)
Comment (donizer) [reply src/index.tsx:5]: It throws an error without importing React or if I import React but don't call it.

Comment (donizer) [reply src/index.tsx]: image| {width:100%}

From different task, but same error.

**Pull request closed without merging**